MaterialLib/MPL/Properties/SaturationDependentSwelling.cpp

```diff
@@ -0,0 +1,157 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "MaterialLib/MPL/Properties/SaturationDependentSwelling.h"
+
+#include <algorithm>
+#include <iostream>
+#include <cmath>
+
+#include "MaterialLib/MPL/Medium.h"
+#include "MathLib/KelvinVector.h"
+#include "ParameterLib/CoordinateSystem.h"
+
+namespace MaterialPropertyLib
+{
+SaturationDependentSwelling::SaturationDependentSwelling(
+    std::array<double, 3> swelling_pressures,
+    std::array<double, 3>
```

Comment:
Is this clang formatted?

Reply:
clang-9.0.1 does it this way, unfortunately.